README.md

```diff
@@ -241,6 +241,67 @@ var pino = require('pino')({browser: {write: {
 }}})
 ```
 
+#### `serialize`: (Boolean | Array)
+
+The serializers provided to `pino` are ignored by default in the browser, including
+the standard serializers provided with Pino. Since the default destination for log
+messages is the console, values such as `Error` objects are enhanced for inspection, 
+which they otherwise wouldn't be if the Error serializer was enabled.
+
+We can turn all serializers on, 
+
+```js
+var pino = require('pino')({
+  browser: {
+    serialize: true
+  }
+})
+```
+
+Or we can selectively enable them via an array:
+
+```js
+var pino = require('pino')({
+  serializers: {
+    custom: myCustomSerializer,
+    another: anotherSerializer
+  },
+  browser: {
+    serialize: ['custom']
+  }
+})
+// following will apply myCustomSerializer to the custom property,
+// but will not apply anotherSerizlier to another key
+pino.info({custom: 'a', another: 'b'})  
+```
+
+When `serialize` is `true` the standard error serializer is also enabled (see https://github.com/pinojs/pino/blob/master/docs/API.md#stdSerializers).
+This is a global serializer which will apply to any `Error` objects passed to the logger methods.
+
+If `serialize` is an array the standard error serializer is also automatically enabled, it can
```

Comment:
Why? The above documentation says that the array value selectively enables the specified serializers. So why would it also enable the error serializer when that wasn't specified?

Reply:
Because the error serializer is global (based on whether it's an instance of `Error`, rather than based on an object key) 

Initially I made is so if you supply an array you could do

`{ serialize: ['stdSerializers.err'] }` 

But then figured it should actually be opt out, because if you enable some serializers, you probably want to have Error serialization as well (because you're opting into using serialization)

I know it's a tricky one, for me this seemed the best set of tradeoffs, you can explicitly disable Error serializing if you want:

`{ serialize: ['!stdSerializers.err', 'someOtherSerializer'] }` 

I think this makes more sense because it provides the most likely desired behaviour with least config